pages/common/git-describe.md

```diff
@@ -0,0 +1,24 @@
+# git describe
+
+> Give an object a human readable name based on an available ref.
+> More information: <https://git-scm.com/docs/git-describe>.
+
+- Create a unique name for the current commit (the name contain: the most recent annotated tag, the number of additional commits and the abbreviated commit hash):
```

Comment:
Perhaps this will read better?
```suggestion
- Create a unique name for the current commit (the name contains the most recent annotated tag, the number of additional commits, and the abbreviated commit hash):
```